doc/release/1.15.0-notes.rst

```diff
@@ -47,64 +65,68 @@ Deprecations
     when called with a string.
 
 * Multidimensional indexing with anything but a tuple is
-  deprecated. This means that code such as ``ind = [slice(None), 0]``,
+  deprecated. This means that code such as ``ind = [slice(None), 0]`` and
   ``arr[[slice(None), 0]]`` should be changed to ``arr[tuple(ind)]``. This is
-  necessary to avoid ambiguity in expressions such as ``arr[[[0, 1], [0, 1]]]``
-  which currently is interpreted as ``arr[array([0, 1]), array([0, 1])]``.
-  In future, this will be interpreted as ``arr[array([[0, 1], [0, 1]])]``.
+  necessary to avoid ambiguity in expressions such as ``arr[[[0, 1], [0, 1]]]``,
+  which is currently interpreted as ``arr[array([0, 1]), array([0, 1])]``.
+  In the future, that will be interpreted as ``arr[array([[0, 1], [0, 1]])]``.
 
 * Direct imports from the following modules is deprecated. All testing related
   imports should come from `numpy.testing`.
+
   * `np.testing.utils`
   * `np.testing.decorators`
   * `np.testing.nosetester`
   * `np.testing.noseclasses`
   * `np.core.umath_tests`
```

Comment:
`umath_tests` cannot be imported from `np.testing`, making the sentence above incorrect. I would explicitly state it has been turned into a private module to indicate that it may change without notice.